reconstruction/rtof/pom.xml

```
@@ -0,0 +1,35 @@
+<?xml version="1.0" encoding="UTF-8"?>
+ <project xmlns="http://maven.apache.org/POM/4.0.0"
+ xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+ xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+  <modelVersion>4.0.0</modelVersion>
+
+  <groupId>org.jlab.clas12.detector</groupId>
+  <artifactId>clas12detector-recoiltof</artifactId>
+  <version>1.0-SNAPSHOT</version>
+  <packaging>jar</packaging>
+
+  <parent>
+    <groupId>org.jlab.clas</groupId>
+    <artifactId>clas12rec</artifactId>
+    <relativePath>../../parent/pom.xml</relativePath>
+    <version>11.1.2-SNAPSHOT</version>
+  </parent>
+
+  <dependencies>
+
+    <dependency>
+      <groupId>org.jlab.clas</groupId>
+      <artifactId>clas-detector</artifactId>
+      <version>11.1.2-SNAPSHOT</version>
+    </dependency>
+
+    <dependency>
+      <groupId>org.jlab.clas</groupId>
+      <artifactId>clas-analysis</artifactId>
+      <version>11.1.2-SNAPSHOT</version>
+    </dependency>
```

and these versions have to match too...
```suggestion
    <dependency>
      <groupId>org.jlab.clas</groupId>
      <artifactId>clas-detector</artifactId>
      <version>13.4.0-SNAPSHOT</version>
    </dependency>

    <dependency>
      <groupId>org.jlab.clas</groupId>
      <artifactId>clas-analysis</artifactId>
      <version>13.4.0-SNAPSHOT</version>
    </dependency>
```